pkg/backends/websockets.go

```diff
@@ -140,8 +140,9 @@ func (b *WebsocketListener) Start(ctx context.Context, wg *sync.WaitGroup) (chan
 	}
 	wg.Add(1)
 	b.server = &http.Server{
-		Addr:    b.address,
-		Handler: mux,
+		Addr:              b.address,
+		Handler:           mux,
+		ReadHeaderTimeout: 10 * time.Second,
```

Comment:
any other reason for this timeout other than tests?

Reply:
Yes, this is a mitigation for the Slowloris attack.